### ISSUE TITLE
Stream http

### ISSUE THREAD

**Issue by dade2906** [sdetweil/MMM-PythonPrint#6]
Hi, i use this module to run in background a stream in particular this https://github.com/raspberrypi/picamera2/blob/main/examples/mjpeg_server.py
This work out of magic mirror but when i run magic mirror the stream not work

**Comment by sdetweil** [comment]
this module captures stdout (from print)

**Comment by dade2906** [comment]
> this module captures stdout (from print)

Yes but i can run all type of python file i think right?

**Comment by sdetweil** [comment]
yes  but you may have to do other steps to link the path to lo python libraries.  or setup python venv.
